iis-common/src/test/java/eu/dnetlib/iis/common/java/jsonworkflownodes/SelectiveTestingConsumer.java

```diff
@@ -41,14 +39,12 @@
  */
 public class SelectiveTestingConsumer implements Process {
 
-    private static final String PROPERTIES_CSV = "expectation_properties_csv";
+    private static final String EXPECTATIONS_CSV = "expectations_csv";
```

Comment:
What would you think of changing this to 'expectation_file_path'? If truth be told, this is what it is.